agcaballero/walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -33,68 +37,126 @@
    */ 
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
     Collection<String> attendees = request.getAttendees();
-    ArrayList<TimeRange> times = new ArrayList<TimeRange>();
-     // minutes of day is + 1 to account for first and last minute of day
-    boolean[] minutes = new boolean[MINUTES_IN_DAY + 1];
+    Collection<String> optionalAttendees = request.getOptionalAttendees();
 
-    for(int i = 0; i < minutes.length; i++) {
-      minutes[i] = true;
-    }
+    // minutes of day is + 1 to account for first and last minute of day
+    Availability[] minutes = new Availability[MINUTES_IN_DAY + 1];
+    // set all spots to default value of all available
+    Arrays.fill(minutes, Availability.ALL_AVAILABLE);
```

Comment:
I like how you replaced the loop with this function! Easier to read!

Reply:
Thank you!